tc-report/_includes/head.html

```diff
@@ -2,7 +2,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>GC Talent | {{ page.title | replace: "<br>", " " | strip_html }}</title>
+  <title>GC Digital Talent | {{ page.title | replace: "<br>", " " | strip_html }}</title>
```

Comment:
I think you'll need to use the `site-title` variable instead of **GC Digital Talent** and then re-generate the site, otherwise the title does not change based on the locale.

Reply:
Makes sense to me!

Ref: [Localize site title](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4468/commits/dfef2fc756db21626f901f5e64db743c6b617b33)